irmin-mirage-git.opam

```diff
@@ -26,3 +25,13 @@ depends: [
 ]
 
 synopsis: "MirageOS-compatible Irmin stores"
+
```

Comment:
Why do we have to pin the dependencies here? Does it work only with a specific commit with no release?

Reply:
I need to remove them yes, I will do soon.

Reply:
Thanks :)